.github/workflows/lint-and-test-charts.yml

```diff
@@ -5,7 +5,7 @@ on: pull_request
 
 jobs:
   lint:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
pin ubuntu version to avoid errors like this 
<img width="959" alt="Screenshot 2025-01-31 at 14 30 24" src="https://github.com/user-attachments/assets/09a74834-fbc6-488d-8a00-ad60b35825ea" />